frontend/src/locales/it.json

```diff
@@ -20,6 +20,9 @@
           }
         }
       },
+      "copyActivityInfoDialog": {
+        "allow": ""
```

Comment:
this prevents the fallback from being displayed.

Reply:
```suggestion
        "allow": "Consenti ora"
```